matter_server/common/custom_clusters.py

```diff
@@ -237,6 +237,132 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: float32 = 0
 
+@dataclass
+class NeoCluster(Cluster, CustomClusterMixin):
+    """Custom (vendor-specific) cluster for Neo - Vendor ID 4991 (0x137F)."""
+
+    id: ClassVar[int] = 0x00125dfc11
+
+    @ChipUtility.classproperty
+    def descriptor(cls) -> ClusterObjectDescriptor:
+        """Return descriptor for this cluster."""
+        return ClusterObjectDescriptor(
+            Fields=[
+                ClusterObjectFieldDescriptor(
+                    Label="wattAccumulated", Tag=0x00125d0021, Type=float32
+                ),
+                ClusterObjectFieldDescriptor(
+                    Label="watt", Tag=0x00125d0023, Type=float32
+                ),
+                ClusterObjectFieldDescriptor(
+                    Label="current", Tag=0x00125d0022, Type=float32
+                ),
+                ClusterObjectFieldDescriptor(
+                    Label="voltage", Tag=0x00125d0024, Type=float32
+                ),
+            ]
+        )
+
+    watt: float32 | None = None
+    wattAccumulated: float32 | None = None
+    voltage: float32 | None = None
+    current: float32 | None = None
+
+    class Attributes:
+        """Attributes for the Neo Cluster."""
+
+        @dataclass
+        class Watt(ClusterAttributeDescriptor, CustomClusterAttributeMixin):
+            """Watt Attribute within the Neo Cluster."""
+
+            should_poll = True
```

Comment:
Does it really need to be polled ? The device is not emitting an attribute update event if the value changes ?

Reply:
If the value changes significantly the device will emit an update almost immediately, however when the value only changes marginally (within a few W) updates will not always be emitted. Therefore I thought it would be useful to also poll this value.

Reply:
OK, well, can you then leave it out for the other attributes maybe ? As polling comes with a bit of overhead so only do it if the device doesn't proactively report it at all (such as Eve)

Reply:
I would suggest that we leave it out for all attributes initialy to limit overhead. If the data is disturbed afterwards because the device emites to infrequently for low value changes, i will commit a change request to set polling to true.

Reply:
sounds good, thanks!

Reply:
I've edited the PR to leave out the should_poll, this is all set now.